src/tools/tools_description.rs

```diff
@@ -421,13 +434,52 @@ impl ToolDict {
     }
 }
 
-pub fn tool_description_list_from_yaml(
+pub async fn get_custom_cmdline_tools(
+    gcx: Arc<ARwLock<GlobalContext>>,
+) -> Result<IndexMap<String, CustomCMDLineTool>, String> {
+    let tconfig = load_customization(gcx.clone(), true).await?;
+    let mut tools_dict = tconfig.custom_cmdline_tools.clone();
+    for (_, tool) in tools_dict.iter_mut() {
+        if tool.runs_in_background {
+            tool.description = format!("{}. Runs in background. Action is start by default", tool.description);
```

Comment:
Why would you add anything to description given by the user

Reply:
Because, model always need to known that tool is running in a background